src/ert/_c_wrappers/job_queue/ert_script.py

```diff
@@ -60,8 +60,6 @@ def initializeAndRun(
         argument_types: List[Type[Any]],
         argument_values: List[str],
     ):
-        self.__failed = False
```

Comment:
The message "Clenup error messages in ert script" has a typo and could be more descriptive of what is happening.